tensorflow/examples/android/src/org/tensorflow/demo/StylizeActivity.java

```diff
@@ -455,78 +444,41 @@ private void setStyle(final ImageSlider slider, final float value) {
     }
   }
 
-  @Override
-  public void onImageAvailable(final ImageReader reader) {
-    Image image = null;
-
-    try {
-      image = reader.acquireLatestImage();
-
-      if (image == null) {
-        return;
-      }
-
-      if (computing) {
-        image.close();
-        return;
-      }
-
-      if (desiredSize != initializedSize) {
-        LOGGER.i(
-            "Initializing at size preview size %dx%d, stylize size %d",
-            previewWidth, previewHeight, desiredSize);
-        rgbBytes = new int[previewWidth * previewHeight];
-        rgbFrameBitmap = Bitmap.createBitmap(previewWidth, previewHeight, Config.ARGB_8888);
-        croppedBitmap = Bitmap.createBitmap(desiredSize, desiredSize, Config.ARGB_8888);
-
-        frameToCropTransform =
-            ImageUtils.getTransformationMatrix(
-                previewWidth, previewHeight,
-                desiredSize, desiredSize,
-                sensorOrientation, true);
-
-        cropToFrameTransform = new Matrix();
-        frameToCropTransform.invert(cropToFrameTransform);
-
-        yuvBytes = new byte[3][];
-
-        intValues = new int[desiredSize * desiredSize];
-        floatValues = new float[desiredSize * desiredSize * 3];
-        initializedSize = desiredSize;
-      }
-
-      computing = true;
+  private void resetPreviewBuffers() {
+    croppedBitmap = Bitmap.createBitmap(desiredSize, desiredSize, Config.ARGB_8888);
 
-      Trace.beginSection("imageAvailable");
+    frameToCropTransform = ImageUtils.getTransformationMatrix(
+        previewWidth, previewHeight,
+        desiredSize, desiredSize,
+        sensorOrientation, true);
 
-      final Plane[] planes = image.getPlanes();
-      fillBytes(planes, yuvBytes);
-
-      final int yRowStride = planes[0].getRowStride();
-      final int uvRowStride = planes[1].getRowStride();
-      final int uvPixelStride = planes[1].getPixelStride();
-
-      ImageUtils.convertYUV420ToARGB8888(
-          yuvBytes[0],
-          yuvBytes[1],
-          yuvBytes[2],
-          previewWidth,
-          previewHeight,
-          yRowStride,
-          uvRowStride,
-          uvPixelStride,
-          rgbBytes);
+    cropToFrameTransform = new Matrix();
+    frameToCropTransform.invert(cropToFrameTransform);
+    yuvBytes = new byte[3][];
+    intValues = new int[desiredSize * desiredSize];
+    floatValues = new float[desiredSize * desiredSize * 3];
+    initializedSize = desiredSize;
+  }
 
-      image.close();
-    } catch (final Exception e) {
-      if (image != null) {
-        image.close();
-      }
-      LOGGER.e(e, "Exception!");
-      Trace.endSection();
-      return;
+  protected void processImageRGBbytes(int[] rgbBytes ) {
+    if (desiredSize != initializedSize) {
+      LOGGER.i(
```

Comment:
Not sure I understand github's behavior here -- is this method actually removed? Why does it still show up here when viewing "all commits", but not when I try to edit/view the file by itself?

@osdamv The log message should be indented 4 spaces here.

@drpngx If I can see all the cumulative commits together in the "files changed" view, do you know why I can't directly edit the latest version of it in the branch here? "View" and "edit" show me a different file than the side-by-side view.